### DISCUSSION TITLE
message: "new row violates row-level security policy for table \"objects\""

### DISCUSSION THREAD

**Discussion by toguvr** [supabase/supabase#2185] [Questions]
i tried avery think... i tried with client anon secre, with service_key, i tried doing an api with multer with all keys, i tried apply policies under bucket and on bucket to authenticated and anon but its didnt worked for me.

**Comment by toguvr** [comment]
and i tried update my bucket to public and private and nothing again..

**Comment by steve-chavez** [comment]
Check this previous discussion: https://github.com/supabase/supabase/discussions/2169

**Reply by toguvr** [reply]
i saw, but didnt worked for me. i dont know why...

**Comment by inian** [comment]
Is this using the hosted version? Are you using the client libraries or making request to the API directly? Can you share a sample snippet? If the service key is being correctly passed in, the RLS error shouldn't happen since service key bypasses RLS

**Reply by toguvr** [reply]
I solved this error changing the policies of my bucket to anyone

**Comment by NavizDev** [comment]
I solved this error: 
https://stackoverflow.com/questions/74302341/supabase-bucket-new-row-violates-row-level-security-policy-for-table-objects/74468038#74468038
